app/services/food_service.rb

```diff
@@ -0,0 +1,17 @@
+class FoodService
+  def self.total_hits(keyword_string)
+    response = conn.get("foods/search?query=#{keyword_string}&dataType=Branded&pageSize=10&")
```

Comment:
nice job adding the pageSize to 10. Here is another way you could break things down:
```
class FoodService
  def self.foods_by_keyword(keyword)
    to_json('/fdc/v1/foods/search', query: keyword)
  end

  private

  def self.to_json(url, params = {})
    response = conn.get(url) do |f|
      f.params = params
      f.params[:pageSize] = 10
    end
    JSON.parse(response.body, symbolize_names: true)
  end

  def self.conn
    Faraday.new(url: 'https://api.nal.usda.gov') do |f|
      f.params[:api_key] = ENV['FOOD_DATA_KEY']
    end
  end
end
```